microservices/first/main.py

```diff
@@ -1,8 +1,10 @@
+# Endpoint documentation: https://github.com/jakejack13/SPEED/blob/SPEED-26-python-database-library/microservices/first_api_doc.pdf
```

Comment:
This link only works for this branch. Once merged, it will stop working. I recommend removing it because the documentation will be easy to find

Reply:
Per Jacob's comment, this should be a markdown file.